contracts/contracts/core/BidderRegistryStorage.sol

```diff
@@ -9,14 +9,13 @@ abstract contract BidderRegistryStorage {
     using FeePayout for FeePayout.Tracker;
 
     /// @dev For improved precision
-    uint256 constant public PRECISION = 1e25;
-    uint256 constant public PERCENT = 100 * PRECISION;
+    uint256 constant public PERCENT = 1e18;
```

Comment:
Why is percent 1e18?

Reply:
To fit into int64 in the oracle decay calculation